LayoutTests/accessibility/mac/selection-value-changes-for-aria-textbox-expected.txt

```diff
@@ -10,7 +10,5 @@ PASS successfullyParsed is true
 TEST COMPLETE
 Successfully received AXSelectedTextChanged
 Successfully received AXSelectedTextChanged
-Successfully received AXSelectedTextChanged
-Successfully received AXSelectedTextChanged
```

Comment:
I recall that when I last wrote a (very similar) version of this patch a few months ago, I was unsure about just rebaseling this test, since it seemed like a potentially risky behavior change. But thinking about this again, maybe it's okay since we're just batching together these notifications, and a real AX user doesn't need to know about each individual selection change/editing operation in a series of selection changes/editing that happen in the same frame.

Reply:
Right. That was my reasoning as well.